src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,19 @@
+.cards {
+  display: grid;
+  justify-content: center;
+  gap: 48px 46px;
```

Comment:
```suggestion
  gap: 46px 48px;
```
First value is the vertical gap
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/06351e4c-2459-4262-a57b-8552d7f7906c)